checkov/common/runners/runner_registry.py

```diff
@@ -90,12 +100,14 @@ def print_reports(
 
         if "cli" in config.output and not config.quiet:
             print(f"{self.banner}\n")
+        file_output_format = config.output[0]
         exit_codes = []
         cli_reports = []
         report_jsons = []
         sarif_reports = []
         junit_reports = []
         cyclonedx_reports = []
+        data_outputs = {'cli': '', 'github_failed_only': ''}
```

Comment:
why not use an empty dict?

Reply:
because the firs use is += so i have to initialize the values before.
but i change to empty dict and check the keys before using '+='

Reply:
@lirshindalman Try `defaultdict` 🙏